### PR TITLE
Correct metadata deleting documentation

### LINKED ISSUE #271: Incorect docs for delete metadata
Delete metadata requests say that they will delete all metadata for an object, but the implemented request actually requires the metadataId and deletes only an individual metadata.

### PR DESCRIPTION
**Pull request by pivnicek**
fixes #271

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -2959,11 +2933,14 @@ Shows all metadata associated with the specified column.
                 }
             ]
 
+## Deleting Metadata [/v2/storage/{objectType}/{objectId}/metadata/{metadata_id}]
```
Comment (odinuv) [inline apiary.apib]: i'd put it in each section, the {objectType} parameter is not in the other API calls either

**Comment by pivnicek** [comment]
Thanks @odinuv , yeah, I just didn't know how to do it elegantly with the different URI than the others in the group.  I'll rewrite, cheers.

**Comment by pivnicek** [comment]
ok, @odinuv I got it, much better, diky.

**Comment by Halama** [comment]
Zavírám, bylo to samé mergnuto v https://github.com/keboola/storage-api-php-client/pull/273

**Pull request closed without merging**